CMakeLists.txt

```diff
@@ -22,12 +22,29 @@ add_definitions(-DPROJECT_CODENAME="${PROJECT_CODENAME}")
 add_definitions(-DPROJECT_QUALITY="${PROJECT_QUALITY}")
 add_definitions(-DPROJECT_SERIES="${PROJECT_SERIES}")
 
-set (LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)" )
-set (LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}" )
-set (PLUGIN_INSTALL_PATH "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}")
-set (INCLUDE_INSTALL_DIR "${CMAKE_INSTALL_INCLUDEDIR}")
-set (DOC_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/${PROJECT_SERIES}")
-set (PLUGIN_SEGMENT_INSTALL_PATH "${CMAKE_INSTALL_SYSCONFDIR}/ambd/plugins.d")
+if(NOT DEFINED LIB_SUFFIX)
```

Comment:
I think the proper way of fixing this would be to fix the automotive-message-broker.pc to generate the prfull pats.  Something like this:

prefix=@CMAKE_INSTALL_PREFIX@
exec_prefix=${prefix}
libdir=${prefix}/@LIB_INSTALL_DIR@
includedir=${prefix}/@INCLUDE_INSTALL_DIR@/amb

Otherwise, if user does not set the cmake variable, it will always be broken.  We can still keep this patch as is to allow for overriding, but we should fix the pc.in files (both in lib/ and in plugins/common